pkg/routeagent_driver/main.go

```diff
@@ -126,9 +131,21 @@ func main() {
 	}
 
 	transitSwitchIP := ovn.NewTransitSwitchIP()
+	submSpec := types.SubmarinerSpecification{}
+	logger.FatalOnError(envconfig.Process("submariner", &submSpec), "Error processing env vars")
 
 	config := &watcher.Config{RestConfig: cfg}
 
+	healthcheckerConfig := &healthchecker.Config{
+		PingInterval:         submSpec.HealthCheckInterval * 60,
```

Comment:
Maybe we should consider running the nodes pinger less frequently than GW2GW pinger